test/SourceKit/DocumentStructure/structure.swift.placeholders.response

```diff
@@ -6,7 +6,7 @@
     {
       key.kind: source.lang.swift.decl.class,
       key.accessibility: source.lang.swift.accessibility.internal,
-      key.name: "<#MyCls#>",
+      key.name: "`<#MyCls#>`",
```

Comment:
This is consistent with how we classify raw identifiers, but admittedly does seem a little odd. It does reflect how placeholders are treated in decl name position though, they are effectively raw identifiers, e.g:

```swift
struct S {
  static func <#foo#>() {}
}
S.<#foo#>() // resolves to <#foo#>
S.<#bar#>() // error: type 'S' has no member '<#bar#>'
```

Any thoughts?

Reply:
Yeah it's a bit of an odd case, really I would prefer if we could just ban editor placeholders in raw identifiers, although that would probably need some evolution discussion.

> I think my preference here would be to only add backticks if the name itself actually has backticks

Trouble is that DeclName doesn't currently have that information, we'd either need to plumb it through, or this decision would need to be left up to the clients

Reply:
I'm happy leaving it for this PR 👍